kilkari/src/main/java/org/motechproject/nms/kilkari/service/impl/SubscriberServiceImpl.java

```diff
@@ -581,6 +585,7 @@ public ChildImportRejection updateRchChildSubscriber(Long msisdn, MctsChild chil
                 if (subscribersByMsisdn.size() == 1 && childUpdate.getMother() != null && subscribersByMsisdn.get(0).getMother() != null && subscribersByMsisdn.get(0).getChild() == null) {
                     //update subscriber with child
                     if (subscribersByMsisdn.get(0).getMother().equals(childUpdate.getMother())) {
+                        childUpdate.setDateOfBirth(dob);
```

Comment:
We should not be updating child DOB here, can you please check why and atleast add a TODO if other tasks have priority